doc/adr/0007-bundesident-url-scheme.md

```diff
@@ -0,0 +1,29 @@
+# 7. Switch from eid:// to bundesident:// scheme
+
+Date: 2022-12-13
+
+## Status
+
+Accepted
+
+## Context
+
+The Technical Guideline TR-03124-1 eID-Client – Part 1 (version 1.4 from 8. October 2021, Chapter 2.2 Full eID-Client) specifies the URL scheme for the ClientURL. 
+Since we are implementing a mobile only approach, described in ADR 5, we need to define the ClientURL as follows: 'eid://127.0.0.1:24727/eID-Client'. 
```

Comment:
This sounds as if we're not sticking to the TR. The chapter 2.2 is only for "Full eID-Clients", but that's not what we are aiming for (for now!). So as long as we're no full eID-Client, we don't need to implement it and shouldn't.